storage/range.go

```diff
@@ -805,10 +841,19 @@ func (r *Range) applyRaftCommand(ctx context.Context, index uint64, originNodeID
 		if !committed {
 			// We didn't commit the batch, but advance the last applied index nonetheless.
 			if err := setAppliedIndex(r.rm.Engine(), r.Desc().RaftID, index); err != nil {
```

Comment:
in this version of the code, it can move back to the main body. Will do that in the next update.